src/Nox.ClientApi.Tests/MsSqlTestProvider.cs

```diff
@@ -0,0 +1,22 @@
+﻿using Microsoft.EntityFrameworkCore;
+using Nox.EntityFramework.SqlServer;
+using Nox.Solution;
+using Nox.Types.EntityFramework.Abstractions;
+
+namespace ClientApi.Tests;
+
+public class MsSqlTestProvider : SqlServerDatabaseProvider
+{
+    public MsSqlTestProvider(string connectionString, IEnumerable<INoxTypeDatabaseConfigurator> configurators) : base(configurators)
+    {
+        ConnectionString = connectionString;
+    }
+
+    public override DbContextOptionsBuilder ConfigureDbContext(DbContextOptionsBuilder optionsBuilder, string applicationName, DatabaseServer dbServer)
+    {
+        return optionsBuilder
+            //.UseLazyLoadingProxies()
```

Comment:
Do we need this commented line?

Reply:
I copied this from "real" provider just in case  to not  forget do the same in test provider.